config/express.js

```diff
@@ -2,7 +2,7 @@
 
 const assert = require("assert");
 const express = require("express");
-const cors = require("cors");
+// const cors = require("cors");
```

Comment:
Ok. By commenting/uncommenting it depending on the environment, you have to update the source several times.

What about an `env` setup for switching it on and off?